### ISSUE TITLE
Section 4.6 Collect Pragmatic-to-Strict Transformation

### ISSUE THREAD

**Issue by samdooley** [w3c/mathml#278]
[Chapter 4](https://w3c.github.io/mathml/#contm) currently spends lots of words on the description of the pragmatic-to-strict transformation algorithm in [section 4.6](https://w3c.github.io/mathml/#contm_p2s) and its subordinate rules, distributed throughout the chapter.

Move the bits of this algorithm from other sections into new subsections of [section 4.6](https://w3c.github.io/mathml/#contm_p2s) that link back to their original locations.

**Comment by davidcarlisle** [comment]
It's true that it takes up some amount of space, and it doesn't affect the set of valid expressions, but the intention was that it was a normative defintion of the meaning as there were comments that the purely prose descriptions in mathml2 were not consistent or clear enough.

that is, some combination of content mathml elements such as

```
<apply><exists/>
  <bvar><ci>x</ci></bvar>
  <domainofapplication><ci>D</ci></domainofapplication>
  <ci>expression-in-x</ci>
</apply>

```

is clearly valid but (the idea was) the intended meaning is given by interpreting this as a shorthand for

```
<bind><csymbol cd="quant1">exists</csymbol>
  <bvar><ci>x</ci></bvar>
  <apply><csymbol cd="logic1">and</csymbol>
    <apply><csymbol cd="set1">in</csymbol><ci>x</ci><ci>D</ci></apply>
    <ci>expression-in-x</ci>
  </apply>
</bind>
```

and strict, having simpler strucure and grounding its terms in openmath, gave some sort of semantic meaning to the original expression.

So if you drop the re-write rules, the two forms are not equivalent by any normative rule and the strict one has a defined mapping to OpenMath and the original one has a meaning described by the words in this spec only.


It's not that OpenMath had any magic bullet for ultimate semantic truth but it did form some fixed point of reference and we will need something similar for intent attributes (whether we end up with wikipedia or our own list or whatever).

So we could drop these rewrites but if they are not normative I'd just drop them altogether they were never intended to be casual descriptive reading, but if they are not normative we probably need much more careful prose description of various combinations of the full content elements if that is to become the normative definition of the "Content" of Content.

**Comment by davidcarlisle** [comment]
@samdooley  that said, it could stay normative but be moved to an appendix which might give more flexibility to make the main chapter 4 be less intimidating and more readable?

**Comment by samdooley** [comment]
Group consensus is to move the bits that describe this algorithm from other sections into subsections of section 4.6 that link back to their original locations.  The expanded section 4.6 could then become an appendix or separate document as appropriate.

**Comment by davidcarlisle** [comment]
@samdooley  looking at the view in your fork, I think this will work out well.

Just one initial comment

The rewrite sections, eg the one for log 

https://samdooley.github.io/mathml/#contm_p2s_log

all have a stylised sentence linking back to the section for that element eg

https://samdooley.github.io/mathml/#contm_log

but there is no reverse link. We could add a new final row in the  `<table class="syntax">`  that appears in each of those sections after **OM Symbols** something like **Strict Rewrite** with a list of links to the appropriate rule ids.

Actually that row could probably be instead of the OM Symbols row (to avod making all those ables bigger) as the OpenMath symbols only really make sense in the context of the rewrite (rewriting to strict being a rewrite to openmath, modulo surface syntax)

**Comment by davidcarlisle** [comment]
this has been merged (#300), still some issues re-eventual location as a section 4.x or an appendix or moved to a different document but I think we can close his then open new issues as needed?
